### PR TITLE
Added the BankTransaction Savina benchmark

### PR DESCRIPTION
**Pull request by EliasC**
This will not work on Mac OSX until #760 is merged!

### CODE REVIEW

File: src/tests/stress/savina/16.BankTransaction/BankTransaction.enc [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    if |args| == 3 then
+      match (args(1).to_int(), args(2).to_int()) with
+        case (Just(a), Just(n)) =>
+          new Teller(a, n)
+          ()
+        end
+        case _ => usage(args(0))
+      end
+    else
+      new Teller(100, 100000)
```
Comment (kaeluka) [inline src/tests/stress/savina/16.BankTransaction/BankTransaction.enc:104]: the default values should be `1000` and `50000` to correspond to the default values of Savina.

**Review by kaeluka — CHANGES_REQUESTED** [review]
I found only one small issue. Once it is addressed, I'll merge this happily.

**Comment by kaeluka** [comment]
This benchmark is, sadly, 10x slower (!!) than most akka benchmarks on my computer.

**Comment by albertnetymk** [comment]
This PR needs to be resumbitted due to publication of originally private repo, "wants to merge 1 commit into `parapluu:development` from `unknown repository`".

**Comment by EliasC** [comment]
> This benchmark is, sadly, 10x slower (!!) than most akka benchmarks on my computer.

I would guess that this is due to heavy use of futures to achieve atomicity.

> This PR needs to be resumbitted due to publication of originally private repo

Reopened as #812

**Pull request closed without merging**